api/accounts/remove.js

```diff
@@ -0,0 +1,20 @@
+module.exports = removeAccount
+
+var deleteAccount = require('../../utils/account/delete')
+
+function removeAccount (state, username, options) {
+  return new Promise(function (resolve, reject) {
+    deleteAccount({
+      couchUrl: state.url,
+      username: username,
+      bearerToken: options.bearerToken,
+      includeProfile: options.include === 'account.profile'
```

Comment:
do you need to mention in [here](http://docs.accountrestapi.apiary.io) that `?include=account.profile` is supported?